docs/build-from-scratch.md

```diff
@@ -71,6 +71,23 @@ Ark's `Makefile` has a convenience target, `all-build`, that builds the followin
 * darwin-amd64
 * windows-amd64
 
+#### Running the Ark server locally
+
+Running the Ark server locally really speeds up the iterative development of server because the newer image does not have to be built into a container and then re-deployed on the cluster.
```

Comment:
Running the Ark server locally can speed up iterative development by eliminating the need to rebuild the Ark server image and redeploy it to the cluster with each change.